src/react/components/styleform/LineTab.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { SvgRadioButton } from 'oskari-ui';
+import { Form, Row } from 'antd';
+
+export const LineTab = (props) => {
+    return (
+        <>
+            <Row>
+                <Form.Item name='lineStyle' label='Line dash' { ...props.formLayout }>
```

Comment:
What does name mean here? It's repeated for all Form.Items with the same value. Is it needed/can it be removed?

Reply:
Name of Form.Item is used for populating values into form.

Reply:
Fixed them now.